### PR TITLE
lowercases ROLE_ defines for consistency (and fixes)

### LINKED ISSUE #36256: Syndie Lavaland Base Turrets Shoot Monkeys
Server-Side Code Modifications: https://tgstation13.org/parsed-logs/basil/server_side_modifications.dm
Round ID: 84647
Server revision compiled on: 2018-03-08
The following pull requests are currently test merged:
#36186: 'this must be the work of a syndicate guardian: adds a new gamemode' by Iamgoofball at commit 0a4f9a
Based off origin/master commit: 84d4b3
Revision: 79b96dd55df6140a0ea79a6c62b9af869a655c9e
Server tools version: /tg/station 13 Server v3.2.1.15
Server tools API version: 3.2.0.2
Current Informational Settings:
Protect Authority Roles From Traitor: 1
Protect Assistant Role From Traitor: 0
Enforce Human Authority: 1
Allow Latejoin Antagonists: 1
Enforce Continuous Rounds: 8 of 19 roundtypes
Allow Midround Antagonists: 6 of 19 roundtypes
All Game Mode Odds:
traitorchan 24.5%
traitor 18.4%
clockwork_cult 8.2%
cult 8.2%
secret_extended 2%
nuclear 16.3%
revolution 12.2%
wizard 10.2%

Easy to see, just pull a monkey out and the turrets shoot it. Admin said they already had Syndie faction.

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
fixes #36256

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
HIGH PRIORITY, CRITICAL, G-B-P!!!!!!!!

**Comment by ExcessiveUseOfCobblestone** [comment]
https://github.com/tgstation/tgstation/pull/36370/files#diff-5bd8b7c7149a06ea6168c7aa527d899eL26

WHY DO YOU PEOPLE DO THIS, STOP STOP STOP

**Comment by praisenarsie** [comment]
it was either shiz or dunc

**Comment by ExcessiveUseOfCobblestone** [comment]
let them eat ca~~ke~~pitalization!

**Comment by Qustinnus** [comment]
in what way does it matter though

File: code/__DEFINES/role_preferences.dm [hunk 1]
```diff
@@ -6,14 +6,14 @@
 
 //These are synced with the Database, if you change the values of the defines
```
Comment (ShizCalev) [inline code/__DEFINES/role_preferences.dm]: reeeee

**Comment by ShizCalev** [comment]
@Qustinnus Maps can't use defines.

**Comment by ShizCalev** [comment]
Just correct the monkey's faction on the map instead.

**Pull request closed without merging**